.github/workflows/test.yml

```diff
@@ -67,6 +64,13 @@ jobs:
           libqhull-dev \
           dpkg-dev
         sudo python3 -m pip install scikit-learn
+        cargo install sccache --version 0.4.2 --locked
```

Comment:
ideally this (and all the apt install above) should be done in a Dockerfile instead of doing this in CI

Reply:
Wouldn't it be quicker to fetch binary archives directly from https://github.com/mozilla/sccache/releases/tag/v0.4.2?

Reply:
cargo install take some time but thats still faster than windows CI anyway. A single line install is better than installing it manually imo.